Fix #3

haozhu233 opened the pull request:
Fix #3 by sourcing Jobs.csv through system.file

haozhu233 committed:
Fix #3

jaredlander commented:
Looks like a good fix, but that needs to be an option in the `param`s so people can specify the file from the command line.  Something like this would be good.  I haven't gotten it to work fully yet but if you do that's great.

```
params:
    JobFile: "`r system.file('examples/Jobs.csv', package='resumer')`"
```

haozhu233 committed:
Add a logic switch

haozhu233 committed:
Update skeleton.Rmd

haozhu233 commented:
Just added a `if` logic. Not a perfect solution but it works now.

haozhu233 committed:
Change the example file name to ".example";

just hope no one will put job info in a system file.. ;)

haozhu233 committed:
Update test cases (Please check)

I don't know why it added a new "on_exit" object to the list. When you have time, please check on this.
